src/probnum/filtsmooth/_gaussfiltsmooth/_extendedkalman.py

```diff
@@ -135,16 +135,16 @@ def __init__(
         non_linear_model,
         mde_atol=1e-5,
         mde_rtol=1e-5,
-        mde_solver="LSODA",
+        mde_solver="RK45",
```

Comment:
the error messages from LSODA are super cryptic (because it wraps fortran code).

Reply:
Makes sense.